crates/polars-ops/src/chunked_array/list/sets.rs

```diff
@@ -410,9 +502,26 @@ pub fn list_set_operation(
         arity::try_binary_unchecked_same_type(
             &a,
             &b,
-            |a, b| array_set_operation(a, b, set_op).map(|arr| arr.boxed()),
+            |a, b| array_set_operation(a, b, set_op).map(|arr| arr.unwrap_left().boxed()),
             false,
             false,
         )
     }
 }
+
+pub fn boolean_list_set_operation(
+    a: &ListChunked,
+    b: &ListChunked,
+    set_op: SetOperation,
+) -> PolarsResult<BooleanChunked> {
+    polars_ensure!(a.len() == b.len() || b.len() == 1 || a.len() == 1, ShapeMismatch: "column lengths don't match");
+
+    let mut a = a.clone();
+    let mut b = b.clone();
+    if a.len() != b.len() {
+        a = a.rechunk();
+        b = b.rechunk();
+    }
+
+    unimplemented!("WIP")
```

Comment:
Don't really understand what should I use here instead of try_binary_unchecked_same_type